src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -333,16 +334,19 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
 export const WorkspaceMenuTrigger = ({ children, canShare, isOwner, setCloningWorkspace, setSharingWorkspace, setDeletingWorkspace }) => h(
   MenuTrigger, {
     closeOnClick: true,
+    'aria-label': 'Workspace menu',
     content: h(Fragment, [
-      h(MenuButton, { onClick: () => setCloningWorkspace(true) }, [makeMenuIcon('copy'), 'Clone']),
+      h(MenuButton, { 'aria-label': 'Workspace copy', onClick: () => setCloningWorkspace(true) }, [makeMenuIcon('copy'), 'Clone']),
```

Comment:
I'm not sure that the word 'workspace' is necessary here, since this is inside something that screenreaders will now (thank you!) read as 'workspace menu':
```suggestion
      h(MenuButton, { onClick: () => setCloningWorkspace(true) }, [makeMenuIcon('copy'), 'Clone']),
```